keps/sig-network/3726-standard-application-protocols/README.md

```diff
@@ -180,6 +181,7 @@ type ServicePort struct {
   //   * 'kubernetes.io/h2c' - HTTP/2 over cleartext as described in https://www.rfc-editor.org/rfc/rfc7540
   //   * 'kubernetes.io/ws'  - WebSocket over cleartext as described in https://www.rfc-editor.org/rfc/rfc6455
   //   * 'kubernetes.io/wss' - WebSocket over TLS as described in https://www.rfc-editor.org/rfc/rfc6455
+  //   * 'kubernetes.io/raw' - Traffic to this port is the protocol value defined by this port's `protocol` field. (ie. TCP/UDP/SCTP)
```

Comment:
Traffic to any port is the protocol defined by `protocol`.  That's tautological.  How about something like:

"Opaque traffic which should not be handled inspected further"

Reply:
That's not quite right either though @thockin. We still want the node OS (or container runtime sandbox, if it's something like gVisor) to process the lower layer stuff. But there's no application protocol on top.

I don't want anyone to misconstrue this as an opt-out from their security appliance and packet / flow inspection.

How about
```suggestion
  //   * 'kubernetes.io/raw' - Opaque application traffic; implementations should not apply special processing
```

Reply:
ok updated. PTAL